### DISCUSSION TITLE
Adding users with existing BudiBase account

### DISCUSSION THREAD

**Discussion by alexandrakubatieva** [Budibase/budibase#11953] [Help] [answered]
Hi, I'm trying to add user with the already existing BudiBase account (the user is also an admin of another unrelated BB app) via `Users` section as it's described in the [documentation](https://docs.budibase.com/docs/adding-users). I'm getting an error: `"There was a problem inviting users. Reason: Email already in use. Please use a different email"`

Does current BB version support this? If so, what might cause this error? Could you confirm that it's likely a tenant issue, same as [here](https://github.com/Budibase/budibase/discussions/9900#discussioncomment-5226080)?

**Comment by ConorWebb96** [comment, marked as answer]
Hey @alexandrakubatieva,

An email can only be added to a single Budibase instance. This error will always be thrown if an email address is already attached to a different tenant. There are ways to get around this e.g. using the `+` email techinique if your email host supports it. (Example `test@test.com` can be already in use but you could use `test+1@test.com` and this would go to the same email address.)

As I can see you've linked my previous response on this, I've opted to keep it brief as its duplicated information.

I hope this explanation helps!

**Reply by alexandrakubatieva** [reply]
@ConorWebb96 Thanks for the prompt reply! Could you explain, if the things are the same for the self-hosted instance? For example, if I deploy two BB instances on my side, will they throw the same error in this case?

**Reply by ConorWebb96** [reply]
Hi @alexandrakubatieva,

You shouldn't run into this issue on self-host as its specific to those installations. The only time this message will likely appear is if you actually do have the same user added into the users area on a per installation basis.

I hope this helps.

**Reply by alexandrakubatieva** [reply]
Got it, thank you!
